### ISSUE TITLE
Trying to train with KITTI dataset

### ISSUE THREAD

**Issue by gxyu-star** [wvangansbeke/Sparse-Depth-Completion#54]
Hello,

Thank you for your excellent work!
I am attempting to train the model with the KITTI dataset, but the generated depth image are not looking correct. Could you please share the detailed and recommended training parameters? I believe there might be an issue with my current settings.
Thanks in advance for your help!

color depth image
<img width="1216" height="352" alt="Image" src="https://github.com/user-attachments/assets/69b66b0f-2595-4923-9ba4-1d50fdacc131" />

**Comment by gxyu-star** [comment]
Some training log including several epochs are showing below

=> loading checkpoint '0'
=> loaded checkpoint '0' (epoch 1)
=>
Args:Namespace(dataset='kitti', nepochs=60, thres=0, start_epoch=1, mod='mod', batch_size=7, val_batch_size=7, learning_rate=0.001, no_cuda=False, evaluate=False, resume='0', nworkers=8, nworkers_val=0, no_dropout=False, subset=None, input_type='rgb', side_selection='', no_tb=True, test_mode=False, pretrained=True, load_external_mod=False, crop_w=1216, crop_h=256, max_depth=85.0, sparse_val=0.0, rotate=False, flip='hflip', rescale=False, normal=False, no_aug=False, save_path='./TrainRetSub_A800/mod_adam_mse_0.001_rgb_batch7_pretrainTrue_wlid0.1_wrgb0.1_wguide0.1_wpred1_patience7_num_samplesNone_multiTrue', data_path='/mnt/train2/depth_estimation/kitti_depth_sub', optimizer='adam', weight_init='kaiming', weight_decay=0, lr_decay=False, niter=50, niter_decay=400, lr_policy='plateau', lr_decay_iters=7, clip_grad_norm=0, gamma=0.5, loss_criterion='mse', print_freq=10000, save_freq=100000, metric='rmse', metric_1='mae', wlid=0.1, wrgb=0.1, wpred=1, wguide=0.1, cudnn=True, gpu_ids='1', multi=True, seed=True, use_disp=False, num_samples=None, world_size=1, dist_url='env://', local_rank=0)

Init model: 'mod'
Number of parameters in model MOD is 2.545M

 => Start EPOCH 2
2025-09-28 14:58:41
./TrainRetSub_A800/mod_adam_mse_0.001_rgb_batch7_pretrainTrue_wlid0.1_wrgb0.1_wguide0.1_wpred1_patience7_num_samplesNone_multiTrue
===> Average RMSE score on training set is 2.2288
===> Average MAE score on training set is 0.8976
=> Start validation set
===> Average RMSE score on validation set is 2.4319
===> Average MAE score on validation set is 1.1061
=> Start selection validation set
===> Average RMSE score on selection set is 2.2028
===> Average MAE score on selection set is 0.9570
===> Last best score was RMSE of 1.8989 in epoch 1
LR plateaued, hence is set to 0.001

 => Start EPOCH 3
2025-09-28 15:58:53
./TrainRetSub_A800/mod_adam_mse_0.001_rgb_batch7_pretrainTrue_wlid0.1_wrgb0.1_wguide0.1_wpred1_patience7_num_samplesNone_multiTrue
===> Average RMSE score on training set is 1.9024
===> Average MAE score on training set is 0.7742
=> Start validation set
===> Average RMSE score on validation set is 2.3383
===> Average MAE score on validation set is 0.9135
=> Start selection validation set
===> Average RMSE score on selection set is 1.6393
===> Average MAE score on selection set is 0.5820
===> Last best score was RMSE of 1.8989 in epoch 1
LR plateaued, hence is set to 0.001
Best model copied

 => Start EPOCH 4
2025-09-28 16:59:05
./TrainRetSub_A800/mod_adam_mse_0.001_rgb_batch7_pretrainTrue_wlid0.1_wrgb0.1_wguide0.1_wpred1_patience7_num_samplesNone_multiTrue
===> Average RMSE score on training set is 1.6790
===> Average MAE score on training set is 0.6668
=> Start validation set
===> Average RMSE score on validation set is 2.4045
===> Average MAE score on validation set is 1.0690
=> Start selection validation set
===> Average RMSE score on selection set is 1.8069
===> Average MAE score on selection set is 0.7920
===> Last best score was RMSE of 1.6393 in epoch 3
LR plateaued, hence is set to 0.001

 => Start EPOCH 5
2025-09-28 17:59:33
./TrainRetSub_A800/mod_adam_mse_0.001_rgb_batch7_pretrainTrue_wlid0.1_wrgb0.1_wguide0.1_wpred1_patience7_num_samplesNone_multiTrue
===> Average RMSE score on training set is 1.5659
===> Average MAE score on training set is 0.6102
=> Start validation set
===> Average RMSE score on validation set is 2.5306
===> Average MAE score on validation set is 1.0330
=> Start selection validation set
===> Average RMSE score on selection set is 1.5690
===> Average MAE score on selection set is 0.6307
===> Last best score was RMSE of 1.6393 in epoch 3
LR plateaued, hence is set to 0.001
Best model copied

**Comment by gxyu-star** [comment]
I set --multi true when I ran test.py and got the normal results
